src/components/structures/MatrixChat.tsx

```diff
@@ -298,7 +299,7 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
         this.themeWatcher.start();
         this.fontWatcher.start();
 
-        this.focusComposer = false;
+        this.focusNext = undefined;
```

Comment:
This is the implicit value, do we really gain anything from this?

Reply:
Nah, I guess not.